src/api.py

```diff
@@ -0,0 +1,220 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+Usage metrics JSON API. Simple web.py based API to a
+PostgreSQL database that runs on port 8080.
+
+usage: python metrics.py
```

Comment:
Minor: I guess this will need to be updated, too. 
`usage: python api.py`

Reply:
done - also updated the list of dependencies